pkg/daemon/util/copybins.go

```diff
@@ -67,5 +67,5 @@ func copyBinary(sourceDir, targetDir, filename string) error {
 		return err
 	}
 
-	return os.Chmod(targetPath, 0755)
+	return os.Chmod(targetPath, 0700) // #nosec binaries requires the permission to execute
```

Comment:
Based on the comment, shouldn't this be `0755`?

Reply:
0700 will be enough for user to execute the binary. But for the group and world to execute this binary then 0755 is needed. Since this is being ran inside the containers I'm still confused figuring out the permissions needed.

Reply:
Ok, I guess I was confused by the comment then.

Reply:
Okay. If the comment is confusing I can change that. @leseb

Reply:
No, it's fine.

Reply:
Cool